specification/resources/resource-manager/Microsoft.Resources/stable/2018-05-01/resources.json

```diff
@@ -34,6 +34,31 @@
     }
```

Comment:
In this file, when we use autorest to check ARM violations, we always have ARMViolations for "TagDetails". The error is that we must have name and type properties for it, and "TagDetails" has extra properties ['tagName', 'count', 'values']. I checked resources.json for 2018-02-01, the definition for "TagDetails" is the same, but there is no such errors in autorest. Do you know how I can address it?

Reply:
@lmazuel I will leave all these suppression to @ifeoluwaokunoren. In this PR, I will fix others, he will send a new PR for suppression.

Reply:
I will let @ravbhatnagar knows this suppression, he or I will send another PR for these suppression rules.